### ISSUE TITLE
Bug: X-Ray trace regression in Envoy image v1.15.0.0-prod 

### ISSUE THREAD

**Issue by lavignes** [aws/aws-app-mesh-roadmap#246]
**Summary**
The X-Ray traces emitted by Envoy 1.15 are different than previous releases.

**Steps to Reproduce**
*You can use https://github.com/aws/aws-app-mesh-examples/tree/master/walkthroughs/howto-ecs-basics as a test application*
* Set `ENVOY_IMAGE` to `840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-envoy:v1.15.0.0-prod`.
* Follow instructions at least through chapter 3 to set up the mesh.
* Observe that:
  * The nodes in the service map are missing the type (segment origin): `AWS::AppMesh::Proxy`.
  * The segment names are the full VirtualNode name: `mesh/howto-ecs-basics/virtualNode/howto-ecs-basics-front-node` instead of `howto-ecs-basics/howto-ecs-basics-front-node`.
  * The [segment documents](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html) are missing the `aws` metadata:
    ```
    "aws": {
      "app_mesh": {
        "mesh_name": "howto-ecs-basics",
        "virtual_node_name": "howto-ecs-basics-front-node"
      }
    }
    ```

![Screenshot from 2020-08-10 17-07-16](https://user-images.githubusercontent.com/1406859/89842957-052b7700-db2c-11ea-8906-e4a743d98f74.png)

**Are you currently working around this issue?**

Using the older Envoy image: `840364872350.dkr.ecr.<region>.amazonaws.com/aws-appmesh-envoy:v1.12.5.0-prod` does not reproduce this behavior.

**Comment by jamsajones** [comment]
We are working on back porting the 1.16 fix to 1.15 App Mesh docker images.

**Comment by jamsajones** [comment]
We have the fix back-ported and will be release 1.15.1 soon.

**Comment by lavignes** [comment]
Since Envoy 1.15.1 is now out: #268. I think we're good to close this. Anyone watching this, feel free to make a new issue or re-open if you're still seeing this after upgrading to 1.15.1
